csrankings.csv

```diff
@@ -9263,7 +9263,7 @@ Mario Lopez,University of Denver,http://portfolio.du.edu/mlopez,NOSCHOLARPAGE
 Mario R. F. Benevides,UFRJ,http://www.cos.ufrj.br/~mario,O3lJvk4AAAAJ
 Mario Schaarschmidt,University of Koblenz-Landau,https://www.uni-koblenz-landau.de/de/koblenz/fb4/ifm/agschaarschmidt/team/schaarschmidt/mario-schaarschmidt,LmkFPYkAAAAJ
 Mario Vento,University of Salerno,http://mivia.unisa.it/people/vento,3PwXGpgAAAAJ
-Mariofanna Milanova,University of Arkansas - Little Rock,http://ualr.academia.edu/MariofannaMilanova,oJr0BfEAAAAJ
+Mariofanna Milanova,University of Arkansas - Little Rock,http://ualr.academia.edu/MariofannaMilanova,n4Uu1lIAAAAJ

```

Comment:
This is my fourth random check and that's a 3/4 rate of incorrect replacement Google Scholar links. I'd be happier if you replaced these with NOSCHOLARPAGE.